### PR TITLE
ADAPT-1758 Continuous prep work

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# NOT READY
- (Edit the above to reflect status)

# Summary
- TL;DR - what's this PR for?

# Needed By (Date)
- When does this need to be merged by?

# Urgency
- How critical is this PR?

# Steps to Test

1. Do this
1. Then this
2. Then this

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)

### CODE REVIEW

File: src/IdentityBar/index.js [hunk 1]
```diff
@@ -1,30 +1,30 @@
 import React from 'react'
 
-export const BrandBar = (props) => {
+export const IdentityBar = (props) => {
 
   // Defaults.
   const defaultClasses = {
```
Comment (sherakama) [inline src/IdentityBar/index.js:6]: I'm not a fan of what I did here with the class arrays. This takes away too much from how clear and easy it is with working in these templates. I want to try another approach where I separate out the variants into separate templates.

Comment (sherakama) [reply src/IdentityBar/index.js:6]: Not to take away from these updates but I am going to try a refactor of this approach that we might be able to discuss on friday.

Comment (yvonnetangsu) [reply src/IdentityBar/index.js:6]: Sounds great! Thanks.

**Comment by yvonnetangsu** [comment]
Closing this old one since we're changing approach

**Pull request closed without merging**